### DISCUSSION TITLE
Mute MPV launching from command line switch / arguments

### DISCUSSION THREAD

**Discussion by pcgeek86** [mpv-player/mpv#15391] [Q&A] [answered]
I typically launch MPV from the Windows "Run" prompt, to monitor security cameras. I'd like these cameras to be muted, rather than having to mute them each time I launch MPV. They pick up a lot of wind noise, which isn't helpful, and typically I just need to see the video.

I realize that I could edit the MPV configuration file, and set the mute option there. Generally, I prefer to specify all my required parameters on the command line, so I can take that command, run it from any computer, and get a similar result.

Does MPV have a command line argument that I can specify to mute the audio at launch time?

**Comment by guidocella** [comment, marked as answer]
https://mpv.io/manual/master/#options-mute

**Reply by pcgeek86** [reply]
Thank you! That's helpful.

```
mpv --mute=yes [rtsp_url]
```

**Reply by guidocella** [reply]
Just `--mute` also works.
